### ISSUE TITLE
Bubble chart text min value

### ISSUE THREAD

**Issue by cescoferraro** [AnSavvides/d3act#29]
The bubble chart needs to have a value bigger than 15 to display text. This value is too random, should not this be a prop?

https://github.com/AnSavvides/d3act/blob/master/src/components/BubbleChart.js#L15

**Comment by sghgh1996** [comment]
I had this problem too. I just added 15 to all my values. It may not be the right way but it helped me.

**Comment by AnSavvides** [comment]
Good point! If you've got time to, a PR to address this would be great!

**Comment by sghgh1996** [comment]
Actually I really like to do that. Do I remove grater than 15 restriction or make it as a prop? I think removing grater than 15 restriction would be good.

**Comment by AnSavvides** [comment]
I believe I set a limit because smaller bubbles having text would have the text spilling over - as long as we can add an ellipsis or something like that to ensure it doesn't spill out, sure thing!
